test/spec/features/selection/SelectionVisualsSpec.js

```diff
@@ -5,22 +5,17 @@ import {
 
 import selectionModule from 'lib/features/selection';
 import modelingModule from 'lib/features/modeling';
+import outlineModule from 'lib/features/outline';
 
 import {
   classes as domClasses,
   query as domQuery
 } from 'min-dom';
 
-import { getBBox } from 'lib/util/Elements';
-
-import {
-  resizeBounds
-} from 'lib/features/resize/ResizeUtil';
-
 
 describe('features/selection/SelectionVisuals', function() {
 
-  beforeEach(bootstrapDiagram({ modules: [ selectionModule, modelingModule ] }));
+  beforeEach(bootstrapDiagram({ modules: [ selectionModule, modelingModule, outlineModule ] }));
```

Comment:
This should no longer be necessary, selection does not depend on outline.

Reply:
technically true, but then we are trying to check the outline on lines [71](https://github.com/bpmn-io/diagram-js/blob/4e22b2e3027ac1447cfae386cbe8c5ea54acfa00/test/spec/features/selection/SelectionVisualsSpec.js#L71) and [95](https://github.com/bpmn-io/diagram-js/blob/4e22b2e3027ac1447cfae386cbe8c5ea54acfa00/test/spec/features/selection/SelectionVisualsSpec.js#L95).
Should those tests be moved to `OutlineSpec` then?

Reply:
I will move what nedds moving to `OutlineSpec` and what is duplicated (I see already its the case) I will simply remove

Reply:
Please add an "integration test" in the outline, verifying the integration of the feature with `selection`.

Regarding the `SelectionVisualsSpec`, consider the [purpose of that component](https://github.com/bpmn-io/diagram-js/blob/65bd3382dfc9fd6a1f7a43091bcc8a0fc1705020/lib/features/selection/SelectionVisuals.js#L40) and whether or not tests are needed for that.

Reply:
(I see you already did, perfect :rocket:).